components/server/src/bitbucket-server/bitbucket-server-repository-provider.ts

```diff
@@ -146,9 +146,8 @@ export class BitbucketServerRepositoryProvider implements RepositoryProvider {
 
     async getUserRepos(user: User): Promise<string[]> {
         try {
-            // TODO: See if there's another api we could use here, such as recent repos for the user
-            // Only grab one page of up to 100 repos
-            const repos = await this.api.getRepos(user, { limit: 100, maxPages: 1, permission: "REPO_READ" });
+            // TODO: implement incremental search
```

Comment:
please try with our bb search first it was unusable before for testing, `New Workspace` screen just hanged when using drop down

Reply:
https://at-bbs-undo-limit.preview.gitpod-dev.com/orgs/join?inviteId=a5e9198d-dc5b-49d9-a26d-071f323dd703